src/core/static-layout.js

```diff
@@ -199,19 +205,22 @@ export function applyStaticLayout(element) {
  */
 export function getEffectiveLayout(element) {
   // Return the pre-existing value if layout has already been applied.
-  const completedLayoutAttr = element.getAttribute('i-amphtml-layout');
-  if (completedLayoutAttr) {
-    return parseLayout(completedLayoutAttr);
+  const completedLayout = parseLayout(element.getAttribute('layout'));
```

Comment:
Why this change? The `i-amphtml-layout`, if set, implies the SSR has happened an no further computation is needed.